aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/CommanConnectivitySteps.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.features;
+
+import com.aws.greengrass.testing.api.device.Device;
+import com.aws.greengrass.testing.api.model.PillboxContext;
+import com.aws.greengrass.testing.platform.Platform;
+import com.aws.greengrass.testing.platform.PlatformResolver;
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+
+import java.io.IOException;
+import javax.inject.Inject;
+
+/**
+ * Checks the connectivity for platfroms.
+ *
+ * @throws IOException {throws IOException}
+ * @throws InterruptedException {throws IInterruptedException}
+ */
+@ScenarioScoped
+public class CommanConnectivitySteps {
+    private final Platform platform;
+
+    @Inject
+    @SuppressWarnings("MissingJavadocMethod")
+    public CommanConnectivitySteps(Platform platform) {
+          this.platform = platform;
```

Comment:
`PlatformResolver` must be injected. Use `resolve() `to get the platform.

Reply:
When it injects the Platform it is already calling resolve so this is good